easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -318,8 +318,29 @@ def configure_step(self, srcdir=None, builddir=None):
 
         (out, _) = run_cmd(command, log_all=True, simple=False)
 
-        return out
-
+        # sanitycheck for the configuration step
+        self.log.info("Checking python paths")
+        with open('CMakeCache.txt', 'r') as file:
+            lines = file.readlines()
+
+        # Search for Python paths in each line
+        for line in lines:
+            if line.startswith('Python3_EXECUTABLE:FILEPATH='):
+                pythonExecPath = line.split('=')[1].strip()
```

Comment:
Small nitpicking suggestion: let's not start using CamelCase for variable names (we only use that for class names), so please use `python_exec_path`, `python_include_path`, `python_library_path`